.github/workflows/draft-release.yml

```diff
@@ -111,97 +108,51 @@ jobs:
-
-            ##  Prerequisites
-
-            The lifecycle runs as a normal user in a series of unprivileged containers. To export images and cache image layers, it requires access to a Docker daemon **or** Docker registry.
-
-            ## Install
-
-            Extract the `.tgz` file and copy the lifecycle binaries into a [build stack base image](https://github.com/buildpack/spec/blob/master/platform.md#stacks). The build image can then be orchestrated by a platform implementation such as the [pack CLI](https://github.com/buildpack/pack) or [tekton](https://github.com/tektoncd/catalog/blob/master/task/buildpacks/0.1/README.md).
-
-            ## Lifecycle Image
-
-            An OCI image containing the lifecycle binaries is available at buildpacksio/lifecycle:${{ env.SHORT_SHA }}.
-      - name: Upload Pre Release Asset - linux
-        if: contains(env.LIFECYCLE_VERSION, 'rc')
-        uses: actions/upload-release-asset@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        with:
-          upload_url: ${{ steps.create_pre_release.outputs.upload_url }}
-          asset_path: ${{ env.ARTIFACT_LINUX_PATH }}
-          asset_name: lifecycle-v${{ env.LIFECYCLE_VERSION }}+linux.x86-64.tgz
-          asset_content_type: application/gzip
-      - name: Upload Pre Release Asset - windows
-        if: contains(env.LIFECYCLE_VERSION, 'rc')
-        uses: actions/upload-release-asset@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        with:
-          upload_url: ${{ steps.create_pre_release.outputs.upload_url }}
-          asset_path: ${{ env.ARTIFACT_WINDOWS_PATH }}
-          asset_name: lifecycle-v${{ env.LIFECYCLE_VERSION }}+windows.x86-64.tgz
-          asset_content_type: application/gzip
-      - name: Create Release
+      - name: Set pre-release kind
+        if: contains(env.LIFECYCLE_VERSION, 'rc') # e.g., 0.99.0-rc.1
+        run: |
+          echo "RELEASE_KIND=pre-release" >> $GITHUB_ENV
+      - name: Set release kind
         if: "!contains(env.LIFECYCLE_VERSION, 'rc')"
-        id: create_release
-        uses: actions/create-release@latest
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        with:
-          tag_name: v${{ env.LIFECYCLE_VERSION }}
-          release_name: lifecycle v${{ env.LIFECYCLE_VERSION }}
-          draft: true
-          prerelease: false
-          body: |
-            # lifecycle v${{ env.LIFECYCLE_VERSION }}
+        run: |
+          echo "RELEASE_KIND=release" >> $GITHUB_ENV
+      - name: Set release body text
+        run: |
+          cat << EOF > body.txt
+          # lifecycle v${{ env.LIFECYCLE_VERSION }}
 
-            Welcome to `v${{ env.LIFECYCLE_VERSION }}`, a **beta** release of the Cloud Native Buildpacks Lifecycle.
+          Welcome to v${{ env.LIFECYCLE_VERSION }}, a **beta** ${{ env.RELEASE_KIND }} of the Cloud Native Buildpacks Lifecycle.
```

Comment:
Sadly this means we'll have to manually add backticks if we want interpolation. If there's a better way to do this in bash, I'd love to hear about it!